mappings/remind.yaml

```diff
@@ -0,0 +1,62 @@
+model: 
+  - REMIND 2.1
```

Comment:
Not sure why you want to use REMIND 3.0-regions for REMIND 2.1? Better to just leave that as it was...

Reply:
As I said before, REMIND region definitions are agnostic in relation to the model version.
The REMIND region acronyms are unique and they will be always possible to be reported in any current or future version of REMIND.
I changed my PR to replace the 3.0 version by the 2.1 as requested if you prefer to keep that way, although I still can't see any reason that would justify to include any version number to it.